scripts/espresso_block_scanner.py

```diff
@@ -0,0 +1,111 @@
+### Simple live Espresso block scanner
+### Scan all blocks looking for transaction on each VM. 
+### For each found transaction, display the VM block containing that transaction, including inclusion proofs for transactions in the Espresso block.
+
+import urllib.request
+import json;
+import time;
+import datetime;
+import sys;
+host="localhost"
+port="50000"
+if len(sys.argv) > 1:
+	host=sys.argv[1]
+if len(sys.argv) > 2:
+	port=sys.argv[2]
+url="http://"+host+":"+port+"/availability/block/"
+
+# Keep track of transaction to avoid showing duplicates
+payloads = set()
+
+# Scan block by block
+block_number = 0
+while True:
```

Comment:
Recommend using `black` to format the code. It can be added via the flake for example  with

```diff
diff --git a/flake.nix b/flake.nix
index 443aa6d42..6c3422af7 100644
--- a/flake.nix
+++ b/flake.nix
@@ -58,7 +58,7 @@
             docker-compose # provides the `docker-compose` command

             # Python
-            (python3.withPackages (ps: with ps; [ ]))
+            (python3.withPackages (ps: with ps; [ black ]))
             jq
```